shiny-drum-machine/index.html

```diff
@@ -47,9 +47,9 @@
 </head>
 <body id='body'>
     <div id='title'>
-        WebAudio Drum Machine <span id='version'>1.1</span>
+        WebAudio Drum Machine <span id='version'>1.2</span>
     </div>
-    <section class='container active' id='pad'>
+    <section class='container' id='pad'>
```

Comment:
Super nit: do we use single quotes for class and id in HTML? What's the official style in Google 3?

Reply:
TIL: Double quotes. Changed the file to use `"`.